### PR TITLE
Comply with Plugin API

### PR DESCRIPTION
**Pull request by chicharr**
Transform in to a plugin following the plugin system API, while supporting old implementation for backwards compatibility

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -136,34 +124,119 @@ export async function initConversionTracking(parent = document, defaultFormConve
...
     .forEach((cefn) => conversionElements[cefn]());
 }
+
+// Add support for Plugin system
+/**
+ * Load the martech configured as non-delayed
+ * @param {*} context should contain at lease sampleRUM object and toCamelCase function
+ */
+export async function loadLazy(document, pluginOptions, context) {
+  initCTInternal(context, document);
```
Comment (ramboz) [inline src/index.js:241]: Is it intended that we hide the `defaultFormConversionName` option now?

**Review by ramboz — APPROVED** [review]
Looks overall good to me. Left a few comments

File: src/index.js (hunk 1 above)
Comment (chicharr) [reply src/index.js:241]: `defaultFormConversionName` is intended to be used only when initializing a fragment that was added to the DOM after initial instrumentation. E.g. when loading a modal fragment with a form when user clicks a certain button

Comment (chicharr) [reply src/index.js:241]: I have updated the documentation

**Pull request merged**